packages/lesswrong/components/users/account/DeleteAccountConfirmationModal.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react';
+import DialogContent from '@material-ui/core/DialogContent';
+import { isFriendlyUI } from '@/themes/forumTheme';
+import { Components, registerComponent } from '@/lib/vulcan-lib';
+import { Link } from '@/lib/reactRouterWrapper';
+import { ACCOUNT_DELETION_COOLING_OFF_DAYS } from '@/lib/collections/users/helpers';
+
+const styles = (theme: ThemeType) => ({
+  dialogPaper: {
+    maxWidth: 500,
+  },
+  close: {
+    position: "absolute",
+    right: 12,
+    top: 12,
+    cursor: "pointer",
+    "& svg": {
+      color: theme.palette.grey[600],
+      width: 20,
+    },
+  },
+  header: {
+    marginTop: 0,
+    fontSize: 24
+  },
+  text: {
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 14,
+    lineHeight: '20px',
+    fontWeight: 500,
+    color: theme.palette.grey[800]
+  },
+  buttonRow: {
+    display: 'flex',
+    gap: '8px',
+    float: 'right',
+  },
+  link: {
+    color: theme.palette.primary.main
+  }
+});
+
+const DeleteAccountConfirmationModal = ({onClose, confirmAction, classes}: {
+  onClose: () => void,
+  confirmAction: () => Promise<void>,
+  classes: ClassesType<typeof styles>,
+}) => {
+  const {LWDialog, EAButton, Loading, Typography} = Components;
+  const [loading, setLoading] = useState(false);
+
+  return (
+    <LWDialog
+      open={true}
+      onClose={onClose}
+      fullWidth
+      maxWidth={isFriendlyUI ? "md" : "sm"}
+      dialogClasses={{ paper: classes.dialogPaper }}
+    >
+      <DialogContent>
+        <Typography variant="display2" className={classes.header}>
+          Confirm account deletion
+        </Typography>
+        <p className={classes.text}>
+          Your account will be deactivated, and you will be unsubscribed from the Forum Digest. You will be able to log
```

Comment:
I think we should be careful to separate "deletion" from "deactivation"
```suggestion
          Your account will be deleted, and you will be unsubscribed from the Forum Digest. You will be able to log
```

Reply:
I deliberately worded it like this to try and get across that:
- First their account will be be deactivated, just as if they had pressed the other button
- Then after 14 days it will be permanently deleted